conversion/data_from_database.go

```diff
@@ -211,7 +211,7 @@ func (dd *DataFromDatabaseImpl) dataFromDatabaseForDataflowMigration(migrationPr
 		return common.TaskResult[*profiles.DataShard]{Result: p, Err: err}
 	}
 	r := common.RunParallelTasksImpl[*profiles.DataShard, *profiles.DataShard]{}
-	_, err = r.RunParallelTasks(sourceProfile.Config.ShardConfigurationDataflow.DataShards, 20, asyncProcessShards, true)
+	_, err = r.RunParallelTasks(sourceProfile.Config.ShardConfigurationDataflow.DataShards, 20, asyncProcessShards, false)
```

Comment:
how do we log / recover the failures for folks to retry ?

Reply:
Currently any exception from a task is logged in STDOUT. 
The logs contain the payload of the API that has failed which can be used to identify the failing shard.

The failing shard(s) will can then be manually retried depending on where the failure happened. For example, if the Dataflow job failed, only the job can be retried, but if the Datastream launch failed, SMT can be rerun with the failed shards(s) to trigger the orchestration for the failed shards again.